images/runtime/python/3.10/base.bullseye.Dockerfile

```diff
@@ -13,6 +13,7 @@ ENV BUILD_NUMBER=${BUILD_NUMBER}
 RUN ./build.sh python /opt/startupcmdgen/startupcmdgen
 
 FROM oryxdevmcr.azurecr.io/private/oryx/oryx-run-base-${DEBIAN_FLAVOR} as main
+ARG DEBIAN_FLAVOR
```

Comment:
If we're adding `DEBIAN_FLAVOR` to this Dockerfile, we should probably drop the `bullseye` from the file name (unless this is a change happening to every Dockerfile independent of what Debian flavors it will actually be built against).

Reply:
I _think_ that the buildRuntimeImageBases is depending on the bullseye to be there in order to find it, right? https://github.com/microsoft/Oryx/blob/main/build/buildRunTimeImageBases.sh#L67-L68

Reply:
Hmm looks that way, so now I'm curious why `DEBIAN_FLAVOR` is needed in this file if we always know what flavor it'll be 🤔 Let's just leave it as-is since it's not bothering anything, just something interesting in this file.

Reply:
I have same question as Cormac for this new added argument `DEBIAN_FLAVOR`. I think when these `base.bullseye.Dockerfile` getting built, the `DEBIAN_FLAVOR=bullseye` environment variable was set up in the image already. The `generateDockerfiles.sh` script will look for any `base.<debianFlavor>.Dockerfile` under versions folder.